shared/models/address_lines_intl.yml

```diff
@@ -0,0 +1,39 @@
+type: object
+style: deepObject
+explode: true
+
+required:
+  - address_line1
+  - address_country
+  - name
+
+oneOf:
+  - required:
+    - address_city
+    - address_state
+  - required:
+    - address_zip
+
+oneOf:
```

Comment:
Incorrect structure which throws a massive error in Prettier. I thought I changed this earlier, by adding `shared/models/address/required_name_company.yml` and `shared/models/address/required_address_fields.yml`? 

Correct:

```
allOf:
  - $ref: "../address/required_name_company.yml"
  - $ref: "../address/required_address_fields.yml"
```